libs/ledger-live-common/src/exchange/platform/transfer/index.ts

```diff
@@ -64,12 +64,33 @@ const fundProviders: Record<
       Buffer.from([9]),
       Buffer.from("Youhodler", "ascii"),
       Buffer.from(
-        "0457123c7248eb61b5554121e9a606a7238c547c02e0ffe908ae7e83641bd47f36e86d6f9a129ffbdaf9237cdb5985e66ac6c9684bea0e4fda89e7d85ac6e6f081",
+        "0471e9a38549562e7e49c2dc079f1350c75b42b5de23104572dbf0d7d2753d1581a246a8bd6e72257f450c023972ad2190c70b468e133648a47e87cce08d17047d",
         "hex"
       ),
     ]),
     signature: Buffer.from(
-      "304402202336e2c71dbaa36f3333bde05da3848db981f906ab70a43583584b4424b651030220402d328768e016fadf535fb6605131cb658553df13d3e7f2c46bcf14847a2f29",
+      "304402207e18ad4540f47a86f7f4d40db3b9ca21a42026b37e404ef528a0e150ce88098e02206c05b27b66bef58e9a056152ef280077a8d7583260cb94c574a3074b2681ff6a",
+      "hex"
+    ),
+    curve: "secp256r1",
+  },
+  uquid: {
+    /**
+     * nameAndPubkey is the concatenation of:
+     * - an empty buffer of the size of the partner name
+     * - a buffer created from the partner name string in ascii encoding
+     * - a buffer created from the hexadecimal version of the partner public key
+     */
+    nameAndPubkey: Buffer.concat([
+      Buffer.from([9]),
+      Buffer.from("Uquid", "ascii"),
+      Buffer.from(
+        "02f96aa3d99d5ec6ce060808fd99c958c17f23e6cb9c1cf81d6419e11677410d04",
```

Comment:
Also, the public key seems way smaller than for other partners, are you sure it's in the correct format?

Reply:
I am not sure but I have managed to sign it on the CAL side so I take it it is fine